repositories/user.js

```diff
@@ -5,9 +5,22 @@ async function getUserByEmail(email) {
 }
 
 async function createUser(userData) {
-  const { email, password, role } = { ...userData }
+  const {
+    email,
+    name,
+    surname,
+    department,
+    salary,
+    stack,
+    password,
+    role
+  } = { ...userData }
 
-  return await User.create({email: email, password: password, role: role})
+  return await User.create({email, name, surname, department, salary, stack, password, role})
```

Comment:
you can simply pass userData here. You don't need to destruct everything and than pas separatly